interface/patient_file/summary/morphine_fragment.php

```diff
@@ -0,0 +1,99 @@
+<?php
```

Comment:
Is this a misplaced script???

Reply:
I don't think so. But should not be a part of this review.

Reply:
Then should remove this script from this PR.

Reply:
File removed from PR